src/test-tree/createTests.ts

```diff
@@ -190,15 +192,15 @@ export class TestFile {
  *
  * @param file_name - name of the harness that is to be verified
  * @param harness_name - name of harness to be verified
- * @param harness_type - True of proof, false if bolero harness
- * @param harness_type - unwind value of the harness
+ * @param proof_boolean - True if proof, false if bolero harness
+ * @param harness_unwind_value - unwind value of the harness (if it exists)
```

Comment:
Is this still needed? Not sure why `harness_unwind_value` is being passed to `captureFailedChecks()`...